packages/api-client/src/api/bundles/index.ts

```diff
@@ -6,12 +6,18 @@ import { ProductsArguments } from '../../types/API';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export default async (context, params: ProductsArguments, customQuery?: CustomQuery) => {
-  const variables: ProductsInput = {
+  const variables: ProductsInput & {
+    attributesPage: number;
```

Comment:
Create special attributes type that the implementor can provide that we can reuse